Description: Fix the location of the javadoc in the documentation
Author: Charles Fry <cfry@debian.org>
Forwarded: not-needed
diff --git a/index.html b/index.html
index 20afa7b..ea5e698 100644
--- a/index.html
+++ b/index.html
@@ -41,7 +41,7 @@
 </p>
 <p>
 The current JavaDoc for the Bouncy Castle APIs can be found
-<a href="javadoc/index.html">here</a>.
+in the corresponding libbc*-java-doc Debian package.
 </p>
 <p>
 Additional documentation on use of the classes can also be found in the <a href="docs/index.html">docs</a> directory.
